docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.6'
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.6.0
+    container_name: elasticsearch
+    environment:
+      - cluster.name=cerebro-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - esdata1:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    networks:
+      - esnet
+  elasticsearch2:
```

Comment:
What do you think about use --scale instead of use 2 instances in docker compose file: https://docs.docker.com/compose/reference/scale/

Reply:
Ok, I'll try it. I don't think it will work because of the cluster settings ES requires, but who knows.

Reply:
I guess I can keep two ES configs in docker-compose, one for master and the other for workers, and scale directly the workers.

Reply:
Check #5a74785, @dmvieira please.